docs/operator-manual/applicationset/Progressive-Syncs.md

```diff
@@ -105,7 +105,163 @@ spec:
       source:
         repoURL: https://github.com/infra-team/cluster-deployments.git
         targetRevision: HEAD
-        path: guestbook/{{.cluster}}
+  `      path: guestbook/{{.cluster}}
```

Comment:
Typo :)

```suggestion
         path: guestbook/{{.cluster}}
```

Reply:
Hahaha the typo is also in the original example that I copy-pasted, will fix both